src/app/shared/components/VueSelect/VueSelect.vue

```diff
@@ -1,32 +1,35 @@
 <template>
-  <div :class="cssClasses">
-    <select
-      :title="placeholder"
-      :name="name"
-      :id="id"
-      v-validate="validation"
-      :multiple="multiple"
-      :required="required"
-      :disabled="disabled"
-      :autocomplete="autocomplete"
-      v-bind="$attrs"
-      v-on="{
-        ...this.$listeners,
-        input: onInput,
-      }"
-    >
-      <option v-for="(option, idx) in selectOptions" :key="idx" :value="option.value" :selected="isSelected(option)">
-        {{ option.label }}
-      </option>
-    </select>
-    <i :class="$style.icon" v-if="!multiple"></i>
-    <span :class="$style.bar"></span>
-    <label :for="id">{{ placeholder }}<sup v-if="required">*</sup></label>
-  </div>
+  <ValidationProvider v-slot="{ errors }" ref="validator" :vid="id" :name="name" :rules="validation">
+    {{ errors }} {{ value }}
```

Comment:
This seems to be left over from debugging, right?

Reply:
upps, yes